airgun/views/host.py

```diff
@@ -206,7 +206,7 @@ class HostsView(BaseLoggedInView, SearchableViewMixin):
         './/table',
         column_widgets={
             0: Checkbox(locator=".//input[@class='host_select_boxes']"),
-            'Name': Text("//a[contains(@href, '/new/hosts')]"),
+            'Name': Text(".//a[contains(@href, '/new/hosts/') and not(contains(@href, 'Insights'))]"),
```

Comment:
Looks functional but it will be good to have only one precise locator if we can achieve that.